test/utils.go

```diff
@@ -43,12 +43,13 @@ func getBoskosProject(resourceType string) *common.Resource {
 	var project *common.Resource
 	err := retry.OnError(
 		wait.Backoff{
-			Duration: time.Second,
+			Duration: time.Minute,
 			Factor:   1.0,
-			Steps:    10,
+			Steps:    30,
```

Comment:
This is 30 minutes. Do we want to wait this long? also should the retry by faster?

Why not do:

10*time.Second

Steps 30

Do we have evidence that the values you are recommending make sense

Reply:
I don't see anywhere suggesting an estimate of how long it takes, but this is the setup using in https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/blob/7485ad00b362a6172c2bceddcdfba8688a8be0e6/test/e2e/utils/project.go#L35. I think we can start with 5 mins and increase if that's not enough.